File: web3rpc/rpc-specs/code-samples/javascript/klay/gas/feeHistory.js
const { JsonRpcProvider } = require("@klaytn/ethers-ext");
(() => {
const provider = new JsonRpcProvider("https://public-en-baobab.klaytn.net");
    const blockCount = '0x10'
    const lastBlock = 'latest'
    const rewardPercentiles = [0.1, 0.2, 0.3]
    provider.klay.feeHistory(blockCount,lastBlock,rewardPercentiles,{}, (err, data, response) => {
        console.log(data);
    });
   
}
)()

File: web3rpc/rpc-specs/code-samples/python/kaia/gas/feeHistory.py
from web3 import Web3
from web3py_ext import extend

host = "https://api.baobab.klaytn.net:8651"

blockCount = "0x10"
lastBlock = "latest"
rewardPercentiles = [0.1, 0.2, 0.3]

w3 = Web3(Web3.HTTPProvider(host))
kaia_response = w3.kaia.fee_history(blockCount, lastBlock, rewardPercentiles)

print(kaia_response)
